lib/video_merger/buffer_queue.ex

```diff
@@ -59,7 +59,7 @@ defmodule Membrane.VideoMerger.BufferQueue do
       {:ok, Enum.reverse(curr), queue}
     else
       {id, [buffer | _rest]} =
-        Enum.min_by(queue, fn {_id, [x | _rest]} -> x.metadata.pts end, &Ratio.lte?/2)
+        Enum.min_by(queue, fn {_id, [x | _rest]} -> x.pts end, &Ratio.lte?/2)
```

Comment:
The pts should be of type `Membrane.Time.t()` - that is an integer, so we can stick to the default comparator